### PR TITLE
New Phishing wizard

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
1. Added Phishing wizard for server version 6.9.
2. Updated "Phishing" incident type to run playbook automatically.
3. Updated Phishing pack dependencies.

## Screenshots
N.A.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [x] 6.9.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ssokolovich** [comment]
@julieschwartz18 @altmannyarden - Opened this PR which will be merged into master ( and **not** [the one we had for tests](https://github.com/demisto/content/pull/19431))
The idea is to reduce as many failures as possible and avoid bugs.

**Comment by altmannyarden** [comment]
I think it worth you will take a look at the wizard as well (@tomer-pan @idovandijk )

File: Packs/Phishing/Wizards/wizard-Phishing.json [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+    "name": "Phishing",
+    "description": "Welcome to the Deployment Wizard!  These steps will guide you through configuring your content pack so you'll have a working use case when the wizard finishes. Make sure to follow the steps in order, you can always continue the wizard where you left off. Learn more about this pack’s deployment.",
+    "dependency_packs": [
+      {
+        "name": "Email Gateways",
+        "min_required": 1,
+        "packs": [
+          {
+            "name": "EWS",
+            "display_name": "EWS Mail (Note: Please install also the EWS Mail Sender Pack)"
```
Comment (julieschwartz18) [inline Packs/Phishing/Wizards/wizard-Phishing.json:15]: Recommend:
 "display_name": "EWS Mail (Make sure you also install the EWS Mail Sender pack.)"

Comment (ssokolovich) [reply Packs/Phishing/Wizards/wizard-Phishing.json]: Done

File: Packs/Phishing/ReleaseNotes/3_3_0.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Incident Types
+**Phishing**
```
Comment (idovandijk) [inline Packs/Phishing/ReleaseNotes/3_3_0.md:3]: Please explain what changed (autorun)

**Review by idovandijk — CHANGES_REQUESTED** [review]
Awesome work Sasha.
Please see my comments on the files and also the following points that came up from our demo today:
- Make sure the changes for the Wizard do not affect the pack negatively < 6.9
- "Single user" integrations - add note that they provide limited functionality
- Email security gateways - the following integrations are used to fetch alerts:
a. Mimecast
b. FireEye
c. Proofpoint 

Please check whether they have additional functionality that is useful for the phishing playbook. If not, let's remove them.

- Need to check if EWS O365 supports on-prem. We can ask Tomer Malache. If it does, then EWS O365 should be higher priority than EWS v2, regardless of telemetry. Otherwise why is the wizard for Phishing v3 and not Phishing v2? Let's debate with Brian if needed.

**Comment by altmannyarden** [comment]
it is, we are using this use case in our rapid breach response playbooks.

File: Packs/Phishing/pack_metadata.json [hunk 3]
```diff
@@ -15,24 +15,219 @@
...
+        "Incident Response"
     ],
     "itemPrefix": [
         "Email",
         "Reported"
     ],
     "useCases": [
-        "Phishing"
+        "Phishing",
+        "Incident Response"
```
[the remark this answers is not in the document]
Comment (ssokolovich) [reply Packs/Phishing/pack_metadata.json:28]: image
Yes it is there :)

**Comment by ssokolovich** [comment]
> Make sure the changes for the Wizard do not affect the pack negatively < 6.9

Should not affect at all. Wizard is only from 6.9 , and the pack metadata dependencies are set to optional.

> "Single user" integrations - add note that they provide limited functionality

Done. 👍 

>Email security gateways - the following integrations are used to fetch alerts:

Mimecast -> is under the `Block Email - Generic` playbook
FireEye and proofpoint  ->  is under the `Block Domain - Generic` playbook

>Please check whether they have additional functionality that is useful for the phishing playbook. If not, let's remove them.

If you see any others that might be irrelevant, I'll check that and remove accordingly. I listed those based on the playbook flow and it's sub playbook (in any way). As you are the expert for phishing pack :) I might miss something... so just have another look and worst case we can remove/add them later on.


>Need to check if EWS O365 supports on-prem. We can ask Tomer Malache. If it does, then EWS O365 should be higher priority than EWS v2, regardless of telemetry. Otherwise why is the wizard for Phishing v3 and not Phishing v2? Let's debate with Brian if needed.

Based on what I saw on our documentation... the `EWS o365` does not support on-prem. 
image
+ Also checked with @DanilVil, `EWS o365` does not support on-prem at all. Means that logically, the best solution (until we will support multiple fetching integrations through the wizard) is to prioritize `EWS v2` with a higher rank than `EWS o365`.

**Review by idovandijk — APPROVED** [review]
All looks good Sasha 👑. You can merge.

I have to mention that big product issue is nagging me here - we are pushing users to adopt the EWS v2 integration without letting them know that O365 would work much better if they're in the cloud. 
We need to be able to display both of them to the users. This can be important for us in the future too with other wizards. I will discuss with Guy Liberman.

**Comment by ssokolovich** [comment]
> All looks good Sasha 👑. You can merge.
> 
> I have to mention that big product issue is nagging me here - we are pushing users to adopt the EWS v2 integration without letting them know that O365 would work much better if they're in the cloud. We need to be able to display both of them to the users. This can be important for us in the future too with other wizards. I will discuss with Guy Liberman.

Can't agree more!!! this limitation should be fixed in the later version of the wizard. @bmillerPaloAlto - FYI

**Pull request merged**